src/components/job-role.jsx

```diff
@@ -15,15 +15,36 @@ class JobRole extends Component {
 		await this.retreiveJobRole()
 	}
 
+	handleDelete = async (jobName) => {
+		try {
+			await deleteJobRole(jobName)
+			console.log("Job deleted")
+			alert("Job role (" + jobName + ") has successfully been deleted")
```

Comment:
Nicer to use a template string here so it would look like `Job role ${jobName} has successfully been deleted`